### ISSUE TITLE
WebUI: Admin cannot change the password of other users

### ISSUE THREAD

**Issue by mmohring** [openSUSE/open-build-service#2804]
If you are a user with Role Admin, you can access /users route with access to creating and modifying users. But one thing is missing: you cannot change another users password. So if a user forgot his pw for example the Admin cannot fix it.

**Comment by mschnitzer** [comment]
@mmohring as a temporary workaround, you can open the rails console and enter the following commands:
```
user = User.find_by(login: "USER_LOGIN")
user.update_password("NEW_PASSWORD")
user.save
```

**Comment by mmohring** [comment]
I do not agree that this is a new feature, because I can remember that I was able to change as Admin a User Password. That is also what one would expect. If the Admin cannot reset a Password, who else should do that ?

**Comment by mmohring** [comment]
I will try to get it working again that the Admin can reset the PW of a normal User as one would expect it.

**Comment by mschnitzer** [comment]
@mmohring since I didn't find this functionality, I thought it was never implemented. I had a conversation (a week ago) with @adrianschroeter and he said that it should be possible. Maybe he can say something about it. I'll remove the `feature` label now and will tag it as a `bug`

**Comment by mmohring** [comment]
Thank you for filing it as a bug.
